### DISCUSSION TITLE
java/1java/1java

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#551] [Announcements]
# java/1java/1java

梦开始的地方 一切就从这里开始吧 JVM JRE JDK JVM是java虚拟机，针对不同系统有不同的实现，常用的为HotSpot VM。 JDK是功能齐全的SDK，包含JRE和一些其他的工具，例如javac，java等。 JRE是Java运行时环境，仅包含Java应用程序运行时的必要环境。

http://localhost:8080/java/1java/1java.html

**Comment by 870138612** [comment]
测试评论
